docs/source/standard.md

```diff
@@ -39,67 +39,92 @@ What the TRE organisation does to ensure information risk is measured and manage
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+#### Risk assessment
+
+| Statement | Guidance | Mandatory Status |
+| --------- | -------- | ---------------- |
+| You have a way to score risk to understand the underlying severity | You have a risk assesment methodology for scoring risks on multiple axes such as impact and likelihood | Mandatory |
+| You have a process for treating risk using additional controls | Risks can be reduced to a level which brings it within agreed levels of appetite | Mandatory |
+| You have an understanding of risk appe*tite | You have an understanding of risk appetite, ownership of risk and ability to accept risk which falls outside of the appetite should that become necessary | Mandatory |
+| You carry out a Data Protection Impact Assessment (DPIA) for all projects requiring a TRE that are working with personal data | a DPIA is a process designed to identify risks arising out of the processing of personal data and to minimise these risks as far and as early as possible | Mandatory |
 
 ### 1.6 Project management
 
-The ability of the TRE organisation to manage projects effectively.
+*The ability of the TRE organisation to manage projects effectively.*
 
-### 1.7 Researcher accreditation
+#### Project onboarding
 
-The ability of the TRE organisation to ensure that people with access to data are identified correctly and they are suitably qualified.
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have checks in place to ensure a project has the legal, financial and ethical requirements in place for the duration of the project | This includes checks that contracts are in place where required, adequate funding is available for the duration of the project, and responsibilities concerning data ownership are understood by all parties | Mandatory |
 
-#### 1.7.1 Training management
+#### Project closure
 
-The ability of the TRE organisation to track and maintain adequate training levels.
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have standard processes in place for the end of a project, that follow all legal requirements and data security best practice | This includes the archiving of quality and log data along with the archiving or deletion of data sets | Mandatory |
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+#### Roles and responsibilities
 
-#### 1.7.2 Training delivery
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| You have clearly defined roles and responsibilities within your TRE organisation for all members | This may include roles such as users, system administrators, system operators, data providers and more. Every member of your TRE organisation should have a pre-defined role with clear powers and responsibilities | Mandatory |
```

Comment:
What do you mean by `member`? Does it include e.g. external researchers outside the organisation operating the TRE?

Reply:
There's a separate "Roles" section later. Can we combine these two?

Reply:
Yeah will need this to be defined in a glossary of terms. It includes anyone associated with the TRE, including internal/external researchers, operators, builders, admins, DPRs and more (catch all term in essence)